### PR TITLE
Contact me form

### LINKED ISSUE #12: Contact me form

### PR DESCRIPTION
**Pull request by supsshinde**
created contact me form successfully
closes #12

### CODE REVIEW

**Comment by shubham-8551** [comment]
@supsshinde 
please resolve merge conflicts.

**Comment by shubham-8551** [comment]
@supsshinde 

Make a form like this  https://arc-i-tech.github.io/contact.html.

**Comment by supsshinde** [comment]
@shubham-8551 
@priyapal2000 
@shubh9060 
Contact me form updated successfully. Review as soon as possible.

File: index.html [hunk 1]
```diff
@@ -11,9 +11,52 @@
...
+                <label for="mob" class="test">Mobile No</label>
+                <input type="number" id="mob" placeholder="Enter mobile No" >
+    
+                <label for="msg">Message Title</label>
+                <input type="text" id="msg" placeholder="Message Title" >
+
+                <label for="Text">Write Message</label>
+                <input type="text1" id="Text" placeholder="Enter Here" >
+
+               <input type="submit" value="Submit">
```
Comment (priyapal2000) [inline index.html:52]: Add validations and submit action.

**Comment by shubham-8551** [comment]
@supsshinde 
please let me know are you working on validation on submit button

File: index.html (hunk 1 above)
Comment (supsshinde) [reply index.html]: Validations added successfully using JavaScript
@shubham-8551 now u can review this task.

**Comment by supsshinde** [comment]
@shubham-8551  please merge #28 PR

**Review by shubham-8551 — CHANGES_REQUESTED** [review]
After **putting only S in all blocks** ...Email and contact number is mismatched showing in red color. **In this case submit button should not work.**

**Comment by supsshinde** [comment]
@shubham-8551 added validations for contact no and email now you can review it.

**Comment by priyapal2000** [comment]
@shubham-8551 
Please Review this PR.

**Comment by shubham-8551** [comment]
In this task no updated needed accepting PR

**Pull request merged**